### DISCUSSION TITLE
M1 iMac 无法更新桌面版

### DISCUSSION THREAD

**Discussion by A-hongchen** [AlistGo/alist#4444] [General]
如图，更新内容太长导致无法点击到最下面的更新确认按钮
<img width="775" alt="image" src="https://github.com/alist-org/alist/assets/71955305/887acddc-5720-47cd-bb2a-f8c5745841c1">

**Comment by xhofe** [comment]
https://github.com/alist-org/desktop-release/issues/14

**Comment by A-hongchen** [comment]
我有看到这个方法，但是不知道为什么我敲回车以后好像是取消或者关闭，我尝试了几次，都没有更新，版本号也没有变化。如你所说，等待你后续的更新修复吧
